package.json

```diff
@@ -37,5 +37,5 @@
     "typescript": "^4.1.3"
   },
   "author": "Nous engineers <engineering@nous.co>",
-  "license": "MIT"
+  "license": "Unlicense"
```

Comment:
Really wanna do this? It's hardly proprietary

Reply:
https://unlicense.org/

`Unlicense` not `UNLICENSED`